tests/test_fields.py

```diff
@@ -17,6 +18,12 @@
 )
 
 
+def field_names(model):
+    """"Return field names for `model`."""
+    fields = model._meta.get_fields()
+    return {field.name for field in fields}
```

Comment:
I would have liked to use `field_names` from [`incuna-test-utils`](https://github.com/incuna/incuna-test-utils/blob/2a1e7484cdb71b4bcbf6b0b4e14acfcae4a03a87/incuna_test_utils/utils.py#L34) but it looks like the `tests` folder from this project and from `incuna-test-utils` are entering in conflict.